Cargo.toml

```diff
@@ -12,9 +12,9 @@ rand_core = "0.5"
 dusk-bytes = "0.1"
 dusk-bls12_381 = {version = "0.6", default-features = false}
 dusk-jubjub = {version = "0.8", default-features = false}
-dusk-poseidon = {version = "0.19", default-features=false}
+dusk-poseidon = { git="https://github.com/dusk-network/poseidon252", branch="vlopes11/update-plonk", default-features = false }
```

Comment:
Use the pre-released version instead of branches so that the Dusk analyzer does not complain please.

Reply:
Fixed